[KIP-37] fix typo and add mintBatch method at KIP37Mintable extension (#47)
klaytn/kips: KIPs/kip-37.md

        string calldata _uri
    ) external returns (bool);

>     /// @notice Mints token and assigns the token according to the variables `_to` and `_quantity`.

Reviewer: ```suggestion
    /// @notice Mints tokens of the specific token type `_id` and assigns the tokens according to the variables `_to` and `_quantity`.
```

    /// @dev Throws if `msg.sender` is not allowed to mint.
    ///   MUST emit an event `TransferSingle`.
    /// @param _id The token id to mint.
    /// @param _to The address that will receive the minted token.
    /// @param _quantity The quantity of token being minted.
    function mint(
        uint256 _id,
        address _to,
        uint256 _quantity
    ) external;

>     /// @notice Mints tokens in a batch and assigns the tokens according to the variables `_to` and `_quantities`.

Reviewer: ```suggestion
    /// @notice Mints tokens in a batch and assigns the tokens according to the variables `_toList` and `_quantities`.
```

>     /// @param _to The address that will receive the minted token.

Reviewer: ```suggestion
    /// @param _to The address that will receive the minted tokens.
```

Reviewer: @kaikookim How do we define 'token'? A token is defined based on creation of mint?

Author: @kjhman21 'token' also includes 'FT', so it would be better to define it with all the tokens minted. I agree with changing 'token' to 'tokens'.

>     /// @param _quantity The quantity of token being minted.

Reviewer: ```suggestion
    /// @param _quantity The quantity of tokens being minted.
```

    /// @dev Throws if `msg.sender` is not allowed to mint.

>     ///   MUST emit an event `TransferSingle` or `TransferBatch`.

Reviewer: ```suggestion
    ///   MUST emit one or more `TransferSingle`events or a single `TransferBatch` event.
```

>     /// @notice Mints tokens in a batch and assigns the tokens according to the variables `_toList` and `_quantities`.

Reviewer: ```suggestion
    /// @notice Mints tokens of the specific token type `_id` in a batch and assigns the tokens according to the variables `_toList` and `_quantities`.
```

>     ///   MUST emit one or more `TransferSingle`events or a single `TransferBatch` event.

Reviewer: ```suggestion
    ///   MUST emit one or more `TransferSingle` events or a single `TransferBatch` event.
```

    /// @notice Mints tokens of the specific token type `_id` and assigns the tokens according to the variables `_to` and `_value`.
    /// @dev Throws if `msg.sender` is not allowed to mint.
    ///   MUST emit an event `TransferSingle`.
    /// @param _id The token id to mint.
    /// @param _to The address that will receive the minted tokens.
    /// @param _value The quantity of tokens being minted.
    function mint(
        uint256 _id,
        address _to,
        uint256 _value
    ) external;

    /// @notice Mints tokens of the specific token type `_id` in a batch and assigns the tokens according to the variables `_toList` and `_values`.
    /// @dev Throws if `msg.sender` is not allowed to mint.
    ///   MUST emit one or more `TransferSingle` events.
    ///   MUST revert if the length of `_toList` is not the same as the length of `_values`.
    /// @param _id The token id to mint.
    /// @param _toList The list of addresses that will receive the minted tokens.
    /// @param _values The list of quantities of tokens being minted.
    function mint(
        uint256 _id,
        address[] calldata _toList,
        uint256[] calldata _values
    ) external;

>     /// @notice Mints multiple KIP37 tokens of the specific token types `_ids` in a batch and assigns the tokens according to the variables `_to` and `_values`.

Reviewer 2: ```suggestion
    /// @notice Mints multiple KIP-37 tokens of the specific token types `_ids` in a batch and assigns the tokens according to the variables `_to` and `_values`.
```
